diff --git a/arch/arm/mach-imx/mach-imx6q.c b/arch/arm/mach-imx/mach-imx6q.c
old mode 100644
new mode 100755
index d02d47c7..b2bc6e19
--- a/arch/arm/mach-imx/mach-imx6q.c
+++ b/arch/arm/mach-imx/mach-imx6q.c
@@ -435,7 +435,7 @@ static void __init imx6q_init_machine(void)
 	imx_anatop_init();
 	imx6q_csi_mux_init();
 	cpu_is_imx6q() ?  imx6q_pm_init() : imx6dl_pm_init();
-	imx6q_mini_pcie_init();//fj_ltls add for pcie from android
+//	imx6q_mini_pcie_init();//fj_ltls add for pcie from android
 }
 
 #define OCOTP_CFG3			0x440
